src/main/java/org/terasology/web/resources/systemStatus/SystemMetadata.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.web.resources.systemStatus;
+
+import oshi.SystemInfo;
+
+public final class SystemMetadata {
```

Comment:
Some quick javadoc would be awesome, at the class level at least, even for fairly obvious classes :-)